project_scrum/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============
+Project Scrum
+=============
+
+Organize your tasks with the Scrum and Sprint methodology.
+
+*Create Sprints to manage your task deadlines*
+*Manage multiple sprint durations with the possibility of creating different Scrum Teams*
+
+Usage
+=====
+This module implements all concepts defined by the scrum project management methodology for IT companies: * Project with sprints, product owner, scrum master * Sprints with reviews, daily meetings, feedbacks * Product backlog * Sprint backlog
+The scrum projects and tasks inherit from the real projects and tasks, so you can continue working on normal tasks that will also include tasks from scrum projects.
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/reporting-engine/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Mohamed Habib Challouf <medhabib@tenovar.com>
+* Samir Guesmi <samir@tenovar.com>
+* George Daramouskas <gdaramouskas@therp.nl>
```

Comment:
probably we should add the original authors here. @vertelab, can you PR what you want to have here (and in the manifest) so that we don't lose your credits?